src/tag_table.rs

```diff
@@ -1592,13 +1592,13 @@ pub const OPEN_TYPE_LANGUAGES: &[LangTag] = &[
     LangTag { language: "zhn", 	tag: Tag::from_bytes(b"ZHA ") }, // Nong Zhuang -> Zhuang
     LangTag { language: "zkb", 	tag: Tag::from_bytes(b"KHA ") }, // Koibal(retired code) -> Khakass
     LangTag { language: "zlj", 	tag: Tag::from_bytes(b"ZHA ") }, // Liujiang Zhuang -> Zhuang
-    LangTag { language: "zlm", 	tag: Tag::from_bytes(b"MLY ") }, // Malay
+    LangTag { language: "zlm", 	tag: Tag::from_bytes(b"MLY ") }, // Malay 
     LangTag { language: "zln", 	tag: Tag::from_bytes(b"ZHA ") }, // Lianshan Zhuang -> Zhuang
     LangTag { language: "zlq", 	tag: Tag::from_bytes(b"ZHA ") }, // Liuqian Zhuang -> Zhuang
     LangTag { language: "zmi", 	tag: Tag::from_bytes(b"MLY ") }, // Negeri Sembilan Malay -> Malay
     LangTag { language: "zmz", 	tag: Tag::from_bytes(b"BAD0") }, // Mbandja -> Banda
     LangTag { language: "znd", 	tag: Tag(0)	            }, // Zande  [family] != Zande
-    LangTag { language: "zne", 	tag: Tag::from_bytes(b"ZND ") }, // Zande
+    LangTag { language: "zne", 	tag: Tag::from_bytes(b"ZND ") }, // Zande 
```

Comment:
Extra space?

Reply:
Just remove them manually for now. In my editor I have "remove trailing whitespaces" feature, so it's being trimmed automatically.